src/api.rs

```diff
@@ -32,31 +32,16 @@ fn fill_meta(span: &Span, env: Option<String>) -> HashMap<String, String> {
     meta
 }
 
-fn fill_metrics(priority: u32) -> HashMap<String, f64> {
+fn fill_metrics() -> HashMap<String, f64> {
     let mut metrics = HashMap::new();
-    metrics.insert("_sampling_priority_v1".to_string(), f64::from(priority));
+    metrics.insert("_sampling_priority_v1".to_string(), 1f64);
```

Comment:
Below you used `f64::from(1)` so it would be nice to be consistent. I think `1_f64` is also an option which is easier to read than `1f64`.